doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc

```diff
@@ -68,7 +75,7 @@ In this example:
 * `realm-public-key` is the public key of the realm you created. You can find the key in the *Keys* tab in the *Realm settings* page of the realm you created in the RH-SSO Admin Console. If you do not provide a value for `realm-public-key`, the server retrieves it automatically.
 * `auth-server-url` is the  URL for the RH-SSO authentication server.
 * `enable-basic-auth` is the  setting to enable basic authentication mechanism, so that the clients can use both token-based and basic authentication approaches to perform the requests.
-* `resource` is the  name for the client that you created.
+* `resource` is the name for the client that you created. If you want to use client roles, use the client resource name that you used when configuring the client in RH-SSO.
```

Comment:
Same as previous comment:

* `resource` is the name for the client that you created. To use client roles, use the client resource name that you used when configuring the client in RH-SSO.